src/eShop.AppHost/Program.cs

```diff
@@ -14,6 +14,9 @@
 var orderDb = postgres.AddDatabase("OrderingDB");
 var webhooksDb = postgres.AddDatabase("WebHooksDB");
 
+var catalogOpenAi = builder.AddOpenAI("catalogOpenAi");
+var chatOpenAi = builder.AddOpenAI("chatOpenAi");
```

Comment:
Why 2 separate ones? I assume we only want 1, and it flows to both projects.

Reply:
Just in case you'd want to use two instances. But we can make the assumption that a single service is used in this app.